src/main/java/nl/ordina/jobcrawler/scrapers/HTMLStructureException.java

```diff
@@ -0,0 +1,7 @@
+package nl.ordina.jobcrawler.scrapers;
+
+public class HTMLStructureException extends Exception {
+    public HTMLStructureException(String s) {
+        super("HTML structure altered in a critical way:" + s);
+    }
+}
```

Comment:
Exceptions should be in the controller/exception package

Reply:
adapted